### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by keneanung**
I added a first readme to the repository, so that the docker repo gets some documentation as well. I hope I covered everything that's of interest for someone using the image.

@vadi2 does this make the purpose, usage and policies of the image clear?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+A docker simple docker image created from the following ingredients:
+- The official `PostgreSQL` image with `plv8` extension installed
+- Latest **released** `fhirbase`
+
+Together, this creates a very specific, yet stable docker image suitable for all docker environments.
+
+### Usage
+
+Simply
```
Comment (vadi2) [inline README.md]: Take away simply - if someone doesn't have it working, it won't be simple, and they'll be frustrated by your vocabulary

File: README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+### Changes versus the original `postgre` and `postgres-plv8` images
+
+1. The database namespace and user are `fhirbase`. That means that all functions and tables created with the initialization script are found in the fhirbase namespace and you should connect as that user.
+2. The `plv8` extension is activated for every connection by default. This may be seen as a security risk. however, since the purpose of this image is to use fhirbase, which needs `plv8`, this is a quality of life change for the developer.
+
+### Update policy
+
+Tags are used to publish new version of the image with the latest `fhirbase` sql. For those releases, the latest `clkao/postgres-plv8:9.4-x` image is used.
+
+If images with volumes of older `fhirbase` installations are started, there is currently no automigration. Instead you'll want to run the patch.sql file of newer releases to get you up to date.
```
Comment (vadi2) [inline README.md]: How?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    $ docker pull dryfhir/fhirbase-postgres-plv8
+
+and you are ready to go. To start the image, you can use 
+
+    $ docker run -d -v fhirbasevolume:/var/lib/postgresql/data --name postgres dryfhir/fhirbase-postgres-plv8
+    
+On first startup, the image will run all needed `CREATE` statements, which may take some time. During this time, the database is not available yet.
+
+Connecting is than as easy as
```
Comment (vadi2) [inline README.md]: Similar-  say something like 'you can now connect with'

**Comment by keneanung** [comment]
Updated the readme accordingly

**Comment by vadi2** [comment]
Thank you!

**Pull request merged**